app/actions/websocket/channel.ts

```diff
@@ -36,6 +37,9 @@ export async function handleChannelCreatedEvent(serverUrl: string, msg: any) {
 
     const {team_id: teamId, channel_id: channelId} = msg.data;
 
+    if (ephemeral_store.creatingChannel) {
```

Comment:
Avoid executing the whole logic of the websocket event if we are creating a channel right now.